src/dc_tools.rs

```diff
@@ -690,14 +709,14 @@ mod tests {
     fn test_dc_create_outgoing_rfc724_mid() {
         // create a normal message-id
         let mid = dc_create_outgoing_rfc724_mid(None, "foo@bar.de");
-        assert!(mid.starts_with("Mr."));
+        assert!(mid.starts_with(CHAT_PREFIX));
         assert!(mid.ends_with("bar.de"));
         assert!(dc_extract_grpid_from_rfc724_mid(mid.as_str()).is_none());
 
         // create a message-id containing a group-id
         let grpid = dc_create_id();
         let mid = dc_create_outgoing_rfc724_mid(Some(&grpid), "foo@bar.de");
-        assert!(mid.starts_with("Gr."));
+        assert!(mid.starts_with(GROUP_PREFIX));
```

Comment:
i think, the same argument as at https://github.com/deltachat/deltachat-core-rust/pull/1516#discussion_r429581849 fits here: we want to make sure, that the outgoing message-id is understood by systems outside. if, at some point, "outside" changes, we can finally adapt this test.

a match with some internal constant, of course, can be tested in addition to that.

Reply:
Let keep both checks.